neptune/new/attributes/series/fetchable_series.py

```diff
@@ -0,0 +1,55 @@
+#
+# Copyright (c) 2020, Neptune Labs Sp. z o.o.
```

Comment:
I don't know if it matters, but maybe it should be 2021 :p

Reply:
haha, you've caught me copy-pasting :)